### DISCUSSION TITLE
Is the feature of a single-file allure report supported in allure-playwright?

### DISCUSSION THREAD

**Discussion by fylim9** [allure-framework/allure2#2197] [Questions & Support] [answered]
I tried to generate a single report using the command: `yarn allure generate --single-file ./allure-results --clean`, but encountered the error of `--single-file does not exist`. So, it got me wondering, is this feature available in Playwright?

Refer to  https://github.com/allure-framework/allure2/pull/2072 for the feature of single-file report

**Comment by fylim9** [comment, marked as answer]
Issue is resolved after upgrading `allure-commandline` to the latest version.

**Reply by JuscaMolaiwa** [reply]
This gotta be the temporary solution.
